UnityProject/Assets/Scripts/Systems/Electricity/Wire/CableCategories/LowVoltageMachineConnector.cs

```diff
@@ -40,6 +44,17 @@ public bool WillInteract(PositionalHandApply interaction, NetworkSide side)
 
 		public void ServerPerformInteraction(PositionalHandApply interaction)
 		{
+			void Remove()
+			{
+				if (removedOnce) return;
+				removedOnce = true;
```

Comment:
This doesn't really matter since it should be destroyed right On server side?

Reply:
It doesn't get destroyed, there can be a long delay where or some sort of hidden error that causes this to stay up on both the server and the client so this is a counter measure to stop duping for players who double click quickly or are desynced.

Reply:
maybe just have a on spawn server hook to reset it, in case it's getting pooled

Reply:
wdym?

Reply:
the object getting poolled, as in it gets reused

Reply:
if it is indeed getting pooled currently, wont that stop this fix because it will be spawned again and will let people spawn another one again?

Reply:
that's only if it's getting instantly spawned in again, but that would be a separate bug

Reply:
done